Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -2761,28 +2761,28 @@ - (void)_loadBackForwardListFromOtherView:(WebView *)otherView
     // type.  (See behavior matrix at the top of WebFramePrivate.)  So we copy all the items
     // in the back forward list, and go to the current one.
 
-    auto& backForward = _private->page->backForward();
-    ASSERT(!backForward.currentItem()); // destination list should be empty
+    CheckedRef backForward = _private->page->backForward();
+    ASSERT(!backForward->currentItem()); // destination list should be empty
 
-    auto& otherBackForward = otherView->_private->page->backForward();
-    if (!otherBackForward.currentItem())
+    CheckedRef otherBackForward = otherView->_private->page->backForward();
+    if (!otherBackForward->currentItem())
         return; // empty back forward list, bail
 
     WebCore::HistoryItem* newItemToGoTo = nullptr;
 
-    int lastItemIndex = otherBackForward.forwardCount();
-    for (int i = -otherBackForward.backCount(); i <= lastItemIndex; ++i) {
+    int lastItemIndex = otherBackForward->forwardCount();
+    for (int i = -otherBackForward->backCount(); i <= lastItemIndex; ++i) {
         if (i == 0) {
             // If this item is showing , save away its current scroll and form state,
             // since that might have changed since loading and it is normally not saved
             // until we leave that page.
             if (auto* localMainFrame = dynamicDowncast<WebCore::LocalFrame>(otherView->_private->page->mainFrame()))
                 localMainFrame->history().saveDocumentAndScrollState();
         }
-        Ref<WebCore::HistoryItem> newItem = otherBackForward.itemAtIndex(i)->copy();
+        Ref<WebCore::HistoryItem> newItem = otherBackForward->itemAtIndex(i)->copy();
```

Comment:
We can just say Ref here, right? No need to specify the type explicitly.

Reply:
Makes sense, done.